src/utils/getOriginalFileForRecord.ts

```diff
@@ -0,0 +1,15 @@
+import { DerivativeType } from '@permanentorg/sdk';
+import type {
+  File,
+  Record,
```

Comment:
Unrelated reminder : We should update the `permanent-sdk` being used. This is now `ArchiveRecord`.

Reply:
Yes indeed! -- that won't happen before the demo, but it will happen (along with a lot of other related changes) as part of the next PR (there's a working branch with those changes already)

Reply:
Okay!